presto-tablestore/src/test/java/com/facebook/presto/tablestore/TestTablestoreFacade.java

```diff
@@ -0,0 +1,1105 @@
+    {
+        Range r = Range.all(INTEGER); // (-∞,+∞)
+        ColumnValueFilter res = transformFilterForColumnV2(PK1_INT, r);
+        assertNull(res);
+
+        r = equal(INTEGER, 1L); // [1,1]
+        res = transformFilterForColumnV2(PK1_INT, r);
+        SingleColumnValueRegexFilter sf = (SingleColumnValueRegexFilter) res;
+        assertNotNull(sf);
+        assertEquals(PK1_INT, sf.getColumnName());
+        assertEquals(1L, sf.getColumnValue().asLong());
+        assertEquals(EQUAL, sf.getOperator());
+
+        r = Range.lessThan(INTEGER, 10L); //(-∞,10)
+        res = transformFilterForColumnV2(PK1_INT, r);
+        sf = (SingleColumnValueRegexFilter) res;
+        assertNotNull(sf);
+        assertEquals(PK1_INT, sf.getColumnName());
+        assertEquals(10L, sf.getColumnValue().asLong());
+        assertEquals(LESS_THAN, sf.getOperator());
+
+        r = Range.lessThanOrEqual(INTEGER, 10L); // (-∞,10]
+        res = transformFilterForColumnV2(PK1_INT, r);
+        sf = (SingleColumnValueRegexFilter) res;
+        assertNotNull(sf);
+        assertEquals(PK1_INT, sf.getColumnName());
+        assertEquals(10L, sf.getColumnValue().asLong());
+        assertEquals(LESS_EQUAL, sf.getOperator());
+
+        r = Range.greaterThanOrEqual(INTEGER, 10L); // [10,+∞)
+        res = transformFilterForColumnV2(PK1_INT, r);
+        sf = (SingleColumnValueRegexFilter) res;
+        assertNotNull(sf);
+        assertEquals(PK1_INT, sf.getColumnName());
+        assertEquals(10L, sf.getColumnValue().asLong());
+        assertEquals(GREATER_EQUAL, sf.getOperator());
+
+        r = Range.greaterThan(INTEGER, 10L); // (10,+∞)
+        res = transformFilterForColumnV2(PK1_INT, r);
+        sf = (SingleColumnValueRegexFilter) res;
+        assertNotNull(sf);
+        assertEquals(PK1_INT, sf.getColumnName());
+        assertEquals(10L, sf.getColumnValue().asLong());
+        assertEquals(GREATER_THAN, sf.getOperator());
+
+        r = Range.range(INTEGER, 1L, true, 2L, false); //[1,2)
+        res = transformFilterForColumnV2(PK1_INT, r);
+        CompositeColumnValueFilter cf = (CompositeColumnValueFilter) res;
+        assertNotNull(cf);
+        assertEquals(LogicOperator.AND, cf.getOperationType());
+        assertEquals(2, cf.getSubFilters().size());
+        SingleColumnValueRegexFilter sf0 = (SingleColumnValueRegexFilter) cf.getSubFilters().get(0);
+        SingleColumnValueRegexFilter sf1 = (SingleColumnValueRegexFilter) cf.getSubFilters().get(1);
+
+        assertEquals(PK1_INT, sf0.getColumnName());
+        assertEquals(1L, sf0.getColumnValue().asLong());
+        assertEquals(GREATER_EQUAL, sf0.getOperator());
+
+        assertEquals(PK1_INT, sf1.getColumnName());
+        assertEquals(2L, sf1.getColumnValue().asLong());
+        assertEquals(LESS_THAN, sf1.getOperator());
+    }
+
+    /**
+     * 类似于：0:(min,0) 1:(0,1) 2:(1,2) 3:(2,3) 4:(3,4) 5:(4,5) 6:(5,6) 7:(6,7) 8:(7,8) 9:(8,max)，第一个是分区，第二个是min，第三个是max
```

Comment:
If the comment here is necessary, could you translate it into English?

Reply:
done